openmc/model/surface_composite.py

```diff
@@ -1837,3 +1837,49 @@ def __init__(self, center_base: Sequence[float], axis: Sequence[float],
 
     def __neg__(self) -> openmc.Region:
         return +self.plane_bottom & -self.plane_top & -self.cone
+
+class Z_Vessel(CompositeSurface):
+    """Vessel as a composite surface parallel to z-axis
+        includes composite surfaces with a ZCylinder and 
+        semi-ellipsoids on top and the bottom of this cylinder.
+        """
```

Comment:
We'll want to include some parameter descriptions in the documentation string here.